src/st/dai/header.c

```diff
@@ -2,12 +2,16 @@
 #include "dai.h"
 
 extern RoomHeader OVL_EXPORT(rooms)[];
-extern s16** OVL_EXPORT(spriteBanks)[];
+extern SpriteParts* OVL_EXPORT(spriteBanks)[];
 extern u_long* OVL_EXPORT(cluts)[];
+#ifdef VERSION_PSP
+extern LayoutEntity* g_pStObjLayoutHorizontal[];
+#else
 extern LayoutEntity* OVL_EXPORT(pStObjLayoutHorizontal)[];
```

Comment:
Does including `pfn_entity_update.h` here allow you to remove the ifdefs?

Reply:
You mean including `pfn_entity_update.h` here instead of `st_init`?
I'll look into it.

Reply:
I think it's just externs so it could be in both

It should handle the conditional define here for you I think

Reply:
So, it looks like Overlay wants `g_pStObjLayoutHorizontal` as a constant, but it is currently defined in `pfn_entity_update.h` as a pointer for `psp` and required as such in the code. `pfn_entity_update.h` can probably be adjusted to work for both the header and existing code, but that would be outside of the scope of this PR.  For now, I've replicated the relevant logic (adjusted for Overlay) from `pfn_entity_update.h` in this file.